tests/cocotb/basic/test/test_hwpe_stream_merge.py

```diff
@@ -0,0 +1,305 @@
+#---------------------------------
+# Copyright 2023 KULeuven
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+# Author: Ryan Antonio (ryan.antonio@esat.kuleuven.be)
+#---------------------------------
+
+#-----------------------------------
+# Importing useful tools 
+#-----------------------------------
+import os
+import yaml
+import random
+import math
+import sys
+
+#-----------------------------------
+# Importing cocotb 
+#-----------------------------------
+import  cocotb
+from    cocotb.triggers       import RisingEdge, Timer
+from    cocotb.clock          import Clock
+from    cocotb_test.simulator import run
+
+#-----------------------------------
+# Importing pytest
+#-----------------------------------
+import  pytest
+
+#-----------------------------------
+# Extracting and setting important variables and paths
+#-----------------------------------
+hwpe_stream_path = os.getenv("HWPE_STREAM_HOME")
+basic_path       = hwpe_stream_path + "/tests/cocotb/basic"
+
+#-----------------------------------
+# Top-level definitions 
+#-----------------------------------
+# Specify top-level module
+toplevel     = 'wrapper_hwpe_stream_merge'
+# Specify python test name that contains the @cocotb.test. Usually the name of this test.
+module       = "test_hwpe_stream_merge"
+# Specify what simulator to use (e.g., verilator, modelsim, icarus)
+simulator    = "verilator"
+# Specify build directory
+sim_build    = basic_path + "/test/sim_build/{}/".format(toplevel)
+# Get YAML source files path
+src_yml_path = hwpe_stream_path + "/src_files.yml"
+
+#-----------------------------------
+# Global parameters for testing
+# TODO: These are modifiable so change whenever needed
```

Comment:
Is this a TODO for you or for the user?

Reply:
TODO for user. Like change what they want to modify.

Reply:
`TODO's` or `FIXME`s are typically directed towards developers and indicate something that needs to be worked on right now or at some point in the future. Here it looks more optional, so a TODO seems kind of out of place here